src/containers/TxDetails.js

```diff
@@ -29,7 +29,14 @@ const mapStateToProps = (state, ownProps) => ({
   txNonce: state.transactions.pendingTransaction.transaction.nonce,
   txGas: state.transactions.pendingTransaction.transaction.gas,
   txGasPrice: state.transactions.pendingTransaction.transaction.gasPrice,
-  txData: state.transactions.pendingTransaction.transaction.data
+  txData: state.transactions.pendingTransaction.transaction.data,
+  isSafe: state.transactions.pendingTransaction.transaction.isSafe,
+  fetchAccountName: (address) => {
```

Comment:
Why do we need this? We already know `txSenderAddress` and `txRecipientAddress` here, why not fetch the names here as well?